### ISSUE TITLE
Unable to compile OC source

### ISSUE THREAD

**Issue by hishamco** [OrchardCMS/OrchardCore#7928]
While I'm working on PR #7927 when I tried to run the unit tests I got the following errors

![Error](https://user-images.githubusercontent.com/3237266/101718501-60598000-3ab2-11eb-9524-ece2c20eeedc.png)

I tried to clean the solution without luck :(

/cc @jtkech @deanmarcussen

**Comment by jtkech** [comment]
Yes i had the same, do a `git clean -xdf` under the root folder of the solution

Then open Visual Studio, wait for the packages to be restored before building, or do a `dotnet restore` from the command line under the cms web app folder

**Comment by hishamco** [comment]
This is wierd it happen to be long time back, but I didn't think it's back again ;)

Thanks Jean-Thierry
